pkg/aws/policies.go

```diff
@@ -35,9 +35,18 @@ import (
 	client "github.com/openshift/rosa/pkg/aws/api_interface"
 	"github.com/openshift/rosa/pkg/aws/tags"
 	"github.com/openshift/rosa/pkg/helper"
+	"github.com/openshift/rosa/pkg/reporter"
 )
 
-var DefaultPrefix = "ManagedOpenShift"
+const (
+	awsManagedPolicyRegexPattern = `^arn:aws:iam::aws:policy/.*$`
+	awsManagedPolicyUrlPrefix    = "https://docs.aws.amazon.com/aws-managed-policy/latest/reference/"
```

Comment:
I wonder how problematic might this be, considering that if AWS changes this url we'll need a new release :thinking:

Reply:
talked with @gdbranco , one option to fix this is to make the URL prefix tweakable in CS side, and CLI can read it from the API when listing the policies. Maybe we can track this in a new issue.
@ciaranRoche wdyt?

Reply:
Would it be worth create a ticket for this, and following up? Use the ticket to get more context as to how often or how likely it is the AWS will change the URL? And if they do would it have a redirect? 

If the juice is worth the squeeze we can make the API changes. But right now I think we are good with this until we get Aaren to get us to rest of the context.

Reply:
https://issues.redhat.com/browse/OCM-10351
issue created to track this